Documentations/set_manufacturing_data.py

```
@@ -0,0 +1,243 @@
+    buffer[4] = year
+    buffer[5] = month
+    buffer[6] = day
+    
+    # Bluetooth name set flag (will be set to 0xFF by firmware)
+    buffer[7] = 0xFF
+    
+    # Secondary info (4 bytes)
+    info_bytes = info.encode('ascii')[:4].ljust(4)
+    buffer[8:12] = info_bytes
+    
+    print(f"Writing secondary serial:")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)